gprofiler/profilers/java.py

```diff
@@ -32,9 +34,15 @@
     run_in_ns,
     run_process,
     touch_path,
+    wait_event,
     write_perf_event_mlock_kb,
 )
 
+NATIVE_FRAMES_REGEX = re.compile(r"^Native frames:[^\n]*\n(.*?)\n\n", re.MULTILINE | re.DOTALL)
```

Comment:
Explain in comment which are these? (can also comment where in HS code this print happens)

Reply:
What do you mean "which are these"?

Reply:
Whenever writing regexes it is much more readable if I have an example line (or multiple lines in this case) so I can compare the "regex code" to the data it should match.

Additionally, a link to the source code is also nice

Reply:
Added.